src/Plugin/ContextReaction/JsonldTypeAlterReaction.php

```diff
@@ -89,9 +89,13 @@ public function buildConfigurationForm(array $form, FormStateInterface $form_sta
     $config = $this->getConfiguration();
     $form['source_field'] = [
       '#type' => 'select',
-      '#title' => $this->t('Source Field'),
+      '#title' => $this->t('Field containing RDF type information'),
```

Comment:
Looks like the JSONLD suite of tests failed due to looking to target that title: https://travis-ci.org/github/Islandora/islandora/jobs/723897394#L1427-L1429.

Reply:
oops! Thanks @jordandukart!